### DISCUSSION TITLE
Error in using node module in production mode : A dynamic link library (DLL) initialization routine failed

### DISCUSSION THREAD

**Discussion by padamjain** [vercel/vercel#5275] [Help]
0

I am trying to build a node package which will be deployed on windows by using "pkg" module. Internally my node app is dependent on one native addon (.node). When i am running this app in development environment using vs code, everything works fine. But when i am publishing this by creating a package, I am getting below error :

pkg/prelude/bootstrap.js:1244
      throw error;
      ^

Error: A dynamic link library (DLL) initialization routine failed.
I have all the dlls and this addon module in the same directory as the exe is. So that is not an issue.

Any help is appriciated.

**Comment by SatishReddy006** [comment]
@padamjain Have you found any solution? If found any solution please share
